changes.go

```diff
@@ -421,6 +421,7 @@ type ChangeInput struct {
 	Author            *AccountInput          `json:"author,omitempty"`
 	Notify            string                 `json:"notify,omitempty"`
 	NotifyDetails     string                 `json:"notify_details,omitempty"`
+	RespFormatOptions []string               `json:"response_format_options,omitempty"`
```

Comment:
```suggestion
	ResponseFormatOptions []string               `json:"response_format_options,omitempty"`
```

Lets make it a bit more explicit.